gst/nnstreamer/tensor_common.c

```diff
@@ -857,6 +857,65 @@ gst_tensors_config_from_structure (GstTensorsConfig * config,
   return TRUE;
 }
 
+/**
+ * @brief Get tensors caps from tensors config and caps of the src pad
+ * @param pad GstPad to check if it supports other/tensor
+ * @param config tensors config structure
+ * @return caps for given config and pad
+ */
+GstCaps *
+gst_tensors_get_caps (GstPad * pad, const GstTensorsConfig * config)
+{
+  GstCaps *caps = NULL;
+
+  if (config->info.num_tensors == 1 && gst_pad_has_tensor_caps (pad)) {
```

Comment:
What is pad has "other/tensor,type=int32" and config is "other/tensor,type=int64"?